cmd/bots/internal/common/environment.go

```diff
@@ -547,6 +560,15 @@ func replaceNodeWithAlias(node string, nodesAlias map[string]string) string {
 	return node
 }
 
+func ReplaceAliasWithNode(alias string, nodes []entities.Node) string {
```

Comment:
btw, this func replaces nothing. It just returns URL if alias matches and says no error if doesn't match.